### PR TITLE
Approved: l10n: zh_TW: update translation

### PR DESCRIPTION
**Pull request by pan93412**
I did the:
  - Make the translation consistent
  - Translate the remaining part (-> 2 untranslated)
  - Also reviewed some translation

Since I can't upload this to Zanata (it said that I don't have the permission to access this), I went uploading this by PR this.

Thanks!

### CODE REVIEW

File: po/zh_TW.po [hunk 1]
```diff
@@ -763,17 +759,17 @@ msgstr "讀取軟連結 %s 時錯誤：%s\n"
...
 #, c-format
 msgid "Symlink points to BuildRoot: %s -> %s\n"
 msgstr "指向 BuildRoot 的符號連結：%s -> %s\n"
 
 #: build/files.c:1331
-#, fuzzy, c-format
+#, c-format
 msgid "Illegal character (0x%x) in filename: %s\n"
-msgstr "非法字符「%c」(0x%x)"
+msgstr "檔名中有非法字元 (0x%x)：%s\n"
```
Comment (zerng07) [inline po/zh_TW.po:772]: 非法 is better changed to 不合規

File: po/zh_TW.po [hunk 2]
```diff
@@ -463,7 +460,7 @@ msgstr "--hash (-h) 可能只能在軟體包安裝或擦除時指定"
 
 #: rpm.c:148
 msgid "--percent may only be specified during package installation and erasure"
-msgstr "-- percent  可能只能在軟體包安裝或擦除時指定"
+msgstr "-- percent 只能在軟體包安裝或擦除時指定"
```
Comment (zerng07) [inline po/zh_TW.po:463]: 抹除 instead of 擦除 for consistence

File: po/zh_TW.po [hunk 3]
```diff
@@ -1359,35 +1355,35 @@ msgstr "列 %d：第二個 %%prep\n"
...
+msgstr "依賴關係符記必須以字母或數字、'_' 或 '/' 為開頭"
 
 #: build/parseReqs.c:57
 msgid "Versioned file name not permitted"
 msgstr "包含版本的檔案名稱不被允許"
 
 #: build/parseReqs.c:222
 msgid "No rich dependencies allowed for this type"
-msgstr "此類型並沒有豐富的相依性。"
+msgstr "此類型並沒有富依賴關係"
```
Comment (zerng07) [inline po/zh_TW.po:1366]: 富依賴性 is not understandable for common readers, better to use 豐富依賴關係

File: po/zh_TW.po [hunk 4]
```diff
@@ -1579,9 +1571,7 @@ msgstr "無法決定策略名稱：%s\n"
 msgid ""
 "'%s' type given with other types in %%semodule %s. Compacting types to "
 "'%s'.\n"
-msgstr ""
-"「%s」類型以及其他類型給出了 %%semodule %s\n"
-"壓縮類型至「%s」\n"
+msgstr "「%s」類型以及其他類型提供了 %%semodule %s。壓縮類型至「%s」。\n"
```
Comment (zerng07) [inline po/zh_TW.po:1574]: 成 is better than 至 here to present the idea of becoming.

File: po/zh_TW.po [hunk 5]
```diff
@@ -1776,46 +1766,44 @@ msgstr "分配新軟體包實體時發生錯誤(%d)\n"
...
-#, fuzzy
 msgid "Rebuilding outdated index databases\n"
-msgstr "無法在 %s 移除舊的資料庫\n"
+msgstr "正在重新組建過時的索引資料庫\n"
 
 #: lib/backend/ndb/rpmidx.c:204
 #, c-format
 msgid "rpmidx: Version mismatch. Expected version: %u. Found version: %u\n"
-msgstr ""
+msgstr "rpmidx：版本不符。期望版本：%u。找到版本：%u\n"
```
Comment (zerng07) [inline po/zh_TW.po:1787]: 預期 is better than 期望

File: po/zh_TW.po [hunk 6]
```diff
@@ -1776,46 +1766,44 @@ msgstr "分配新軟體包實體時發生錯誤(%d)\n"
...
 
 #: lib/backend/ndb/rpmpkg.c:125
 #, c-format
 msgid "rpmpkg: Version mismatch. Expected version: %u. Found version: %u\n"
-msgstr ""
+msgstr "rpmpkg：版本不符。期望版本：%u。找到版本：%u\n"
```
Comment (zerng07) [inline po/zh_TW.po:1792]: 預期 is better than 期望

File: po/zh_TW.po [hunk 7]
```diff
@@ -1776,46 +1766,44 @@ msgstr "分配新軟體包實體時發生錯誤(%d)\n"
...
 #: lib/backend/ndb/rpmpkg.c:499
 msgid "rpmpkg: detected non-zero blob, trying auto repair\n"
-msgstr ""
+msgstr "rpmpkg：偵測到非 0 的 Blob，嘗試自動修復\n"
 
 #: lib/backend/ndb/rpmxdb.c:237
 #, c-format
 msgid "rpmxdb: Version mismatch. Expected version: %u. Found version: %u\n"
-msgstr ""
+msgstr "rpmxdb：版本不符。期望版本：%u。找到版本：%u\n"
```
Comment (zerng07) [inline po/zh_TW.po:1801]: 預期 is better than 期望

File: po/zh_TW.po [hunk 8]
```diff
@@ -1933,11 +1921,11 @@ msgstr "檔案"
 #: lib/header.c:301
 #, c-format
 msgid "tag[%d]: BAD, tag %d type %d offset %d count %d len %d"
-msgstr "標籤 [%d]：壞的，標籤 %d 類型 %d  偏差 %d 數值 %d 長度 %d"
+msgstr "標籤 [%d]：無效，標籤 %d 類型 %d 偏移 %d 數值 %d 長度 %d"
 
 #: lib/header.c:971
 msgid "hdr load: BAD"
-msgstr "hdr_load：壞的"
+msgstr "hdr_load：無效"
```
Comment (zerng07) [inline po/zh_TW.po:1928]: 損壞 better than 無效

File: po/zh_TW.po [hunk 9]
```diff
@@ -1946,60 +1934,60 @@ msgstr "區域：沒有標籤"
...
 #: lib/header.c:1945
 #, c-format
 msgid "sigh pad(%zd): BAD, read %zd bytes"
-msgstr "sigh_pad(%zd)：壞的，讀取 %zd 位元組"
+msgstr "sigh_pad(%zd)：無效，讀取 %zd 位元組"
 
 #: lib/header.c:1958
 msgid "signature "
-msgstr "簽名"
+msgstr "簽署 "
```
Comment (zerng07) [inline po/zh_TW.po:1990]: verb as 簽署, noun as 簽章

File: po/zh_TW.po [hunk 10]
```diff
@@ -2078,19 +2066,19 @@ msgid "array iterator used with different sized arrays"
 msgstr "用於不同大小陣列的陣列迭代器"
 
 #: lib/package.c:315
-#, fuzzy, c-format
+#, c-format
 msgid "RPM v3 packages are deprecated: %s\n"
-msgstr "第 %d 行：%s 重複：%s\n"
+msgstr "RPM v3 軟體包已過時：%s\n"
```
Comment (zerng07) [inline po/zh_TW.po:2071]: 過時 should be 棄用

File: po/zh_TW.po [hunk 11]
```diff
@@ -2144,27 +2132,27 @@ msgstr "<FILE:…>"
...
 
 #: lib/poptALL.c:233
 msgid "don't verify database header(s) when retrieved"
-msgstr "擷取時不校驗資料庫表頭"
+msgstr "擷取時不核驗資料庫表頭"
 
 #: lib/poptALL.c:235
 msgid "don't verify package signature(s)"
-msgstr "不校驗軟體包簽署"
+msgstr "不核驗軟體包簽署"
```
Comment (zerng07) [inline po/zh_TW.po:2155]: 簽章

File: po/zh_TW.po [hunk 12]
```diff
@@ -2461,79 +2449,79 @@ msgstr "重新安裝軟體包"
...
 
 #: lib/poptQV.c:103
 msgid "rpm verify mode"
-msgstr "rpm 校驗模式"
+msgstr "rpm 核驗模式"
 
 #: lib/poptQV.c:105 lib/poptQV.c:107
 msgid "query/verify the package(s) which require a dependency"
-msgstr "查詢/校驗需要某些相依軟體包的軟體包"
+msgstr "查詢/核驗需要某些相依軟體包的軟體包"
```
Comment (zerng07) [inline po/zh_TW.po:2500]: 依賴軟體包

File: po/zh_TW.po [hunk 13]
```diff
@@ -2461,79 +2449,79 @@ msgstr "重新安裝軟體包"
...
 
 #: lib/poptQV.c:109
 msgid "query/verify the package(s) which obsolete a dependency"
-msgstr ""
+msgstr "查詢或核驗棄用某個依賴關係的軟體包"
 
 #: lib/poptQV.c:111
 msgid "query/verify the package(s) which provide a dependency"
-msgstr "查詢/校驗提供某些相依軟體包的軟體包"
+msgstr "查詢/核驗提供某些相依軟體包的軟體包"
```
Comment (zerng07) [inline po/zh_TW.po:2508]: 依賴軟體包

File: po/zh_TW.po [hunk 14]
```diff
@@ -3021,81 +3009,81 @@ msgstr "支援 POSIX.1e 檔案功能"
...
 
 #: lib/rpmds.c:1248
 msgid "support files larger than 4GB"
-msgstr "檔案支援大於 4GB"
+msgstr "支援大於 4GB 的檔案"
 
 #: lib/rpmds.c:1251
 msgid "support for rich dependencies."
-msgstr "支援富相依性"
+msgstr "支援富依賴關係。"
```
Comment (zerng07) [inline po/zh_TW.po:3028]: 豐富依賴關係

File: po/zh_TW.po [hunk 15]
```diff
@@ -3021,81 +3009,81 @@ msgstr "支援 POSIX.1e 檔案功能"
...
 #: lib/rpmds.c:1258
 msgid "package payload can be compressed using zstd."
-msgstr "軟體包有效負荷可以使用 zstd 壓縮。"
+msgstr "軟體包酬載可以使用 zstd 壓縮。"
 
 #: lib/rpmds.c:1411
 #, c-format
 msgid "Unknown rich dependency op '%.*s'"
-msgstr "未知的富相依性操作「%.*s」"
+msgstr "未知的富依賴關係操作「%.*s」"
```
Comment (zerng07) [inline po/zh_TW.po:3041]: 豐富依賴關係

File: po/zh_TW.po [hunk 16]
```diff
@@ -3591,17 +3579,17 @@ msgstr "%s 標籤 %u：無效的大小 %u"
 #: lib/rpmvs.c:197
 #, c-format
 msgid "%s tag %u: invalid OpenPGP signature"
-msgstr "%s 標籤 %u：無效的 OpenGPG 簽名"
+msgstr "%s 標籤 %u：無效的 OpenGPG 簽署"
```
Comment (zerng07) [inline po/zh_TW.po:3582]: 簽章

File: po/zh_TW.po [hunk 17]
```diff
@@ -3614,7 +3602,7 @@ msgstr "%s%s"
 
 #: lib/rpmvs.c:281
 msgid "signature"
-msgstr "簽名"
+msgstr "簽署"
```
Comment (zerng07) [inline po/zh_TW.po:3605]: 簽章

File: po/zh_TW.po [hunk 18]
```diff
@@ -3630,19 +3618,19 @@ msgstr "表頭 "
...
+msgstr "酬載 "
 
 #: lib/signature.c:213
 msgid "Unable to reload signature header.\n"
 msgstr "無法重新載入簽署表頭。\n"
 
 #: lib/transaction.c:1272
 msgid "no signature"
-msgstr ""
+msgstr "沒有簽署"
```
Comment (zerng07) [inline po/zh_TW.po:3629]: 簽章

File: po/zh_TW.po [hunk 19]
```diff
@@ -3630,19 +3618,19 @@ msgstr "表頭 "
...
 
 #: lib/transaction.c:1272
 msgid "no digest"
-msgstr ""
+msgstr "沒有 digest"
```
Comment (zerng07) [inline po/zh_TW.po:3633]: 摘要

File: po/zh_TW.po [hunk 20]
```diff
@@ -3668,7 +3656,7 @@ msgstr "遺漏      %c %s"
 #: lib/verify.c:440
 #, c-format
 msgid "Unsatisfied dependencies for %s:\n"
-msgstr "%s 的不滿足相依性：\n"
+msgstr "%s 的不滿足依賴關係：\n"
```
Comment (zerng07) [inline po/zh_TW.po:3659]: 「的」是綴字

File: po/zh_TW.po [hunk 21]
```diff
@@ -3690,102 +3678,90 @@ msgid "Failed to register fork handler: %m\n"
...
+msgstr "解析 || 時有語法錯誤"
 
 #: rpmio/expression.c:370
 msgid "macro expansion returned a bare word, please use \"...\""
 msgstr ""
 
 #: rpmio/expression.c:372
 msgid "macro expansion did not return an integer"
-msgstr ""
+msgstr "巨集展開未回傳數字"
```
Comment (zerng07) [inline po/zh_TW.po:3698]: 整數 instead of 數字

File: po/zh_TW.po [hunk 22]
```diff
@@ -3690,102 +3678,90 @@ msgid "Failed to register fork handler: %m\n"
...
-#, fuzzy
 msgid "parse error in expression"
-msgstr "表述式解析錯誤\n"
+msgstr "表述式解析錯誤"
 
 #: rpmio/expression.c:447
-#, fuzzy
 msgid "unmatched ("
-msgstr "不符合的 (\n"
+msgstr "未配對的 ("
```
Comment (zerng07) [inline po/zh_TW.po:3719]: 不成對的 (，看起來是者小括號沒有 match 成對。

File: po/zh_TW.po [hunk 23]
```diff
@@ -3690,102 +3678,90 @@ msgid "Failed to register fork handler: %m\n"
...
-#, fuzzy
 msgid "- only on numbers"
-msgstr "- 只能用於數字\n"
+msgstr "- 只能用於數字"
 
 #: rpmio/expression.c:489
-#, fuzzy
 msgid "unexpected end of expression"
-msgstr "| 預期於表述式的結尾"
+msgstr "表述式非預期結束"
```
Comment (zerng07) [inline po/zh_TW.po:3727]: 未預期結束

File: po/zh_TW.po [hunk 24]
```diff
@@ -3835,13 +3811,13 @@ msgstr "在 %2$s(%3$s) 中有未知的選項 %1$c\n"
...
 msgid "no such macro: '%s'\n"
-msgstr ""
+msgstr "無此巨集：「%s」\n"
 
 #: rpmio/macro.c:1390
 msgid ""
 "Too many levels of recursion in macro expansion. It is likely caused by "
 "recursive macro declaration.\n"
-msgstr "巨集附加元件中太多遞迴階級。這可能是因為遞迴巨集聲明而造成。\n"
+msgstr "巨集展開的遞迴階級更多。這可能是因為遞迴巨集聲明所導致的。\n"
```
Comment (zerng07) [inline po/zh_TW.po:3820]: 階層

File: po/zh_TW.po [hunk 25]
```diff
@@ -3854,14 +3830,12 @@ msgid "A %% is followed by an unparseable macro\n"
...
-#, fuzzy
 msgid "argument expected"
-msgstr "未預期的 ]"
+msgstr "預期是引數"
 
 #: rpmio/macro.c:1488
-#, fuzzy
 msgid "unexpected argument"
-msgstr "未預期的 ]"
+msgstr "預期不是引數"
```
Comment (zerng07) [inline po/zh_TW.po:3838]: 未預期引數

File: po/zh_TW.po [hunk 26]
```diff
@@ -3905,7 +3879,7 @@ msgstr "警告："
 
 #: rpmio/rpmlog.c:269
 msgid "Error writing to log"
-msgstr ""
+msgstr "寫入至記錄檔時發生錯誤"
```
Comment (zerng07) [inline po/zh_TW.po:3882]: 紀錄為名詞，記錄為動詞

File: po/zh_TW.po [hunk 27]
```diff
@@ -3993,7 +3967,7 @@ msgstr "%s：Fflush 失敗： %s\n"
 
 #: sign/rpmgensig.c:203
 msgid "Unsupported PGP signature\n"
-msgstr "不支援的 PGP 簽名\n"
+msgstr "不支援的 PGP 簽署\n"
```
Comment (zerng07) [inline po/zh_TW.po:3970]: 簽章

**Comment by pmatilai** [comment]
Our translations have been in a bit of a limbo with Zanata going out of business and being broken, but we finally have a new home for them: https://translate.fedoraproject.org/projects/rpm/
Please submit your translation to Fedora Weblate, we can't handle (doesn't scale) individual translations updated via GH pull-requests. 

Terribly sorry about all this extra trouble, and thank you for your contribution!

**Pull request closed without merging**